File: TsinghuaNet.UWP/Converters/FluxLimitConverter.vb
Public Class FluxStringConverter
    Implements IValueConverter

    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Return $"{CLng(CDbl(value))} GB"
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Throw New NotImplementedException()
    End Function
End Class

Public Class FluxLimitConverter
    Implements IValueConverter

    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Return CType(value, ByteSize).GigaBytes
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Return ByteSize.FromGigaBytes(value)
    End Function
End Class

File: TsinghuaNet.XF/TsinghuaNet.XF/Converters/FluxLimitConverter.cs
using System;
using System.Globalization;
using TsinghuaNet.Models;
using Xamarin.Forms;

namespace TsinghuaNet.XF.Converters
{
    class FluxLimitConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
            => ((ByteSize)value).GigaBytes;

        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
            => ByteSize.FromGigaBytes((double)value);
    }
}
